test/infrastructure/docker/e2e/docker_suite_test.go

```diff
@@ -113,19 +113,12 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
+	// Dump the logs of the providers before deleting them.
 	Expect(writeLogs(mgmt, "capi-system", "capi-controller-manager", logPath)).To(Succeed())
 	Expect(writeLogs(mgmt, "capi-kubeadm-bootstrap-system", "capi-kubeadm-bootstrap-controller-manager", logPath)).To(Succeed())
 	Expect(writeLogs(mgmt, "capi-kubeadm-control-plane-system", "capi-kubeadm-control-plane-controller-manager", logPath)).To(Succeed())
 	Expect(writeLogs(mgmt, "capd-system", "capd-controller-manager", logPath)).To(Succeed())
```

Comment:
Might be good to dump the webhook logs here as well, but can be handled in followup :)

Reply:
don't tell anyone but there are no webhooks for capd 🙈

Reply:
I only mention this because I was trying to debug things for an issue filed against CAPA (whether webhooks the conversion webhooks were deployed), and it would have been great to have the logs available to see from prow rather than having to spin a cluster up locally. 

The CAPD e2es would have been able to answer it if the question arose for cluster-api core webhooks :)

Reply:
ahh yeah for sure. I'll file an issue to capture that, good point! thanks